virt/virt-4-12-release-notes.adoc

```diff
@@ -122,6 +118,9 @@ The SVVP Certification applies to:
 [id="virt-4-12-web-new"]
 === Web console
 
+//CNV-18453 Live migration with shared storage for work intensive workloads
+* You can create and manage live migration policies in a single location on the *Virtualization > MigrationPolicies* page.
```

Comment:
Hi :) 
does it make sense to remove the first two words "You can" and just leave everything else? "Create and manage ..."

Reply:
We normally say "You can do x" when referring to a new feature. Otherwise, it sounds too much like an order. :-)  
It should have this wording so that it fits with the other new feature RNs.

Reply:
Thanks for explaining, that's sounds logical :)